targets/simple_switch_grpc/m4/ax_cxx_compile_stdcxx_17.m4

```diff
@@ -0,0 +1,35 @@
+# =============================================================================
```

Comment:
I think you also need https://www.gnu.org/software/autoconf-archive/ax_cxx_compile_stdcxx.html in this m4 directory?